src/main/java/de/thomas_oster/liblasercut/drivers/LaserToolsTechnicsCutter.java

```diff
@@ -782,14 +778,7 @@ private double circle(PrintStream out, Point center, double resolution) throws I
   /**
    * cut a smooth curve with given interpolation points and interpolation speeds
    *
-   * @param out
-   * @param x coordinates, starting with currentX
-   * @param y coordinates, starting with currentY
-   * @param endSpeed end speed at given coordinate (in percent, relative to
```

Comment:
Yes, I removed it because the documented parameters do not match the method signature at all:
```java
private double curveWithKnownSpeed(PrintStream out, ArrayList<PointWithSpeed> points, double resolution) throws IOException
```